src/browser/modules/Frame/FrameTitlebar.tsx

```diff
@@ -264,34 +267,30 @@ function FrameTitlebar(props: FrameTitleBarProps) {
   const history = (frame.history || []).slice(1)
   return (
     <StyledFrameTitleBar>
-      <FeatureToggle
-        name={reusableFrame}
-        on={
-          <FrameTitleEditorContainer data-testid="frameCommand">
-            <Monaco
-              history={history}
-              useDb={frame.useDb}
-              enableMultiStatementMode={true}
-              id={`editor-${frame.id}`}
-              bus={props.bus}
-              onChange={setEditorValue}
-              onExecute={run}
-              value={editorValue}
-              ref={editorRef}
-              toggleFullscreen={props.fullscreenToggle}
-            />
-          </FrameTitleEditorContainer>
-        }
-        off={
-          <StyledFrameCommand
-            selectedDb={frame.useDb}
-            onClick={() => props.onTitlebarClick(editorValue)}
-            data-testid="frameCommand"
-          >
-            <DottedLineHover>{editorValue}</DottedLineHover>
-          </StyledFrameCommand>
-        }
-      />
+      {renderEditor ? (
+        <FrameTitleEditorContainer data-testid="frameCommand">
+          <Monaco
+            history={history}
+            useDb={frame.useDb}
+            enableMultiStatementMode={true}
+            id={`editor-${frame.id}`}
+            bus={props.bus}
+            onChange={setEditorValue}
+            onExecute={run}
+            value={editorValue}
+            ref={editorRef}
+            toggleFullscreen={props.fullscreenToggle}
+          />
+        </FrameTitleEditorContainer>
+      ) : (
```

Comment:
Looks like this PR includes changes from the title bar folding PR

Reply:
Whoops I'll reopen a new one